pep-9999.rst

```diff
@@ -0,0 +1,341 @@
+PEP: 9999
+Title: Lazy Imports
+Author: Germán Méndez Bravo <german.mb@gmail.com>, Carl Meyer <carl@oddbird.net>
+Sponsor: Barry Warsaw <barry@python.org>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 29-Apr-2022
+Python-Version: 3.12
+
+Abstract
+========
+
+This PEP proposes an opt-in experimental feature to transparently defer the
```

Comment:
Perhaps remove "opt-in experimental" from the PEP?   It gives the impression that the feature won't be a standard part of Python.  I think it will (i.e. it may start out as experimental, but eventually it will just be how Python works, even if only under certain circumstances).  Also, the rest of the text will explain how to opt-in or -out.